### DISCUSSION TITLE
Re-activating tool after temporary disable

### DISCUSSION THREAD

**Discussion by bodawui** [burkybang/SuiteAdvanced-Feedback#150] [Q&A] [answered]
Hey, I temporary disabled the plugin on the account but now I cannot re-activate it the icon is gone and in the popup from chrome plugin I cannot see any option :-/

**Comment by burkybang** [comment, marked as answer]
This is a known bug, and I can't figure out what causes it. I think it's some kind of caching issue with Chrome. Try using the Ear Cache button at the bottom of SuiteAdvanced settings. If that doesn't work, try uninstalling and reinstalling SuiteAdvanced.
